scripts/preloadHTTPSE.js

```diff
@@ -36,81 +35,124 @@ const buildDataFiles = () => {
-    'PJ_Media.xml': 'mixed content on https://pjmedia.com/instapundit/',
-    'Slashdot.xml': 'redirect loop on mobile slashdot.org',
-    'Delta.com.xml': 'https://delta.com does not redirect to https://www.delta.com',
-    'Cargo.xml': 'breaks cargocollective.com',
-    'TMZ.com.xml': 'breaks www.tmz.com',
-    'BusinessInsider.xml': 'breaks http://www.businessinsider.com/silicon-valley-100-2016-6?op=0',
-    'Tesco.xml': 'breaks tesco.com due to CSP mismatch',
-    'Vodafone.ie.xml': 'breaks pagination on http://shop.vodafone.ie/shop/phonesAndPlans/phonesAndPlansHome.jsp?subPage=phones&planFilter=onAccount',
-    'IDownloadBlog.xml': 'breaks http://www.idownloadblog.com/',
-    'EBay_static.com.xml': 'breaks suggested product image previews',
-    'Cisco.xml': 'breaks http://www.cisco.com/c/m/en_us/training-events/events-webinars/techwise-tv/listings.html',
-    'GQ.xml': 'mixed content on gq.com',
-    'Where_2_Get_It.xml': 'maps missing on http://us.coopertire.com/Customer-Care/Dealer-Locator.aspx?form=locator_search&addressline=92346',
-    'Thompson_Hotels.com.xml': 'missing stylesheets on http://www.thompsonhotels.com/'
+    'Nike.com (partial)': 'breaks nikeplus.com',
+    'PJ Media': 'mixed content on https://pjmedia.com/instapundit/',
+    'Slashdot.org (partial)': 'redirect loop on mobile slashdot.org',
+    'Delta.com (partial)': 'https://delta.com does not redirect to https://www.delta.com',
+    'Cargo (partial)': 'breaks cargocollective.com',
+    'TMZ.com': 'breaks www.tmz.com',
+    'BusinessInsider.com (partial)': 'breaks http://www.businessinsider.com/silicon-valley-100-2016-6?op=0',
+    'Tesco (partial)': 'breaks tesco.com due to CSP mismatch',
+    'Vodafone.ie (partial)': 'breaks pagination on http://shop.vodafone.ie/shop/phonesAndPlans/phonesAndPlansHome.jsp?subPage=phones&planFilter=onAccount',
+    'iDownloadBlog (partial)': 'breaks http://www.idownloadblog.com/',
+    'Cisco.com (partial)': 'breaks http://www.cisco.com/c/m/en_us/training-events/events-webinars/techwise-tv/listings.html',
+    'GQ.com (partial)': 'mixed content on gq.com',
+    'Where 2 Get It (partial)': 'maps missing on http://us.coopertire.com/Customer-Care/Dealer-Locator.aspx?form=locator_search&addressline=92346',
+    'Thompson Hotels.com (partial)': 'missing stylesheets on http://www.thompsonhotels.com/'
   }
 
-  const rulesets = JSON.parse(fs.readFileSync('./https-everywhere/chrome/content/rulesets.json', 'utf8'))
+  const rulesets = JSON.parse(fs.readFileSync('./https-everywhere/rules/default.rulesets', 'utf8'))
 
-  // Convert XML rules to JSON
-  for (let id in rulesets.rulesetStrings) {
-    let contents = rulesets.rulesetStrings[id]
-    parseString(contents, function (err, result) {
-      if (err) {
-        throw new Error('FATAL: error parsing XML: ' + contents)
+  let jsonOutput = {
+    'rulesetStrings': [],
+    'targets': {}
+  }
+
+  for (const ruleset of rulesets) {
+    if (!ruleset.default_off && !ruleset.platform) {
```

Comment:
the original code include `default_off` and `platform` ruleset in `httpse.json`, which is gone now.